src/main/java/io/dashboardhub/pipelinedashboard/domain/User.java

```diff
@@ -23,10 +26,10 @@
     private String username;
 
     private String name;
+    private String email;
 
-//    @ManyToMany
-//    @JoinTable(name="users_projects")
-//    private Set<Project> projects;
+    @OneToMany(cascade = CascadeType.ALL, fetch = FetchType.EAGER)
```

Comment:
Not an issue if I change it to the `lastLoggedIn` field in the **User** class and then *Envers* to audit the changes.